pytorch_toolkit/image_classification/README.md

```diff
@@ -0,0 +1,29 @@
+# Image Classification
+
+## Pre-trained models
+
+This repo contains scripts and tutorials for image classification models training.
+
+* [Custom image classification](model_templates/custom-classification/readme.md) - models that are trained on the ImageNet dataset and suitable for transfer learning.
+
+## Setup
+
+### Prerequisites
+
+* Ubuntu\* 16.04
+* Python\* >=3.6
+* PyTorch\* 1.6.0
+* OpenVINO™ 2020.4 with Python API
+* deep-object-reid (../../external/deep-object-reid)
+
+### Installation
+
+1. Create virtual environment and build deep-object-reid:
```

Comment:
where is a build step?

Reply:
Indeed, it's not visible from there. Removed the mention of it.